pkg/agent/stream/client.go

```diff
@@ -157,10 +167,13 @@ func NewStreamer(
 	host, port := getWatchServiceHostPort(cfg)
 
 	watchCfg := &wm.Config{
-		Host:        host,
-		Port:        uint32(port),
-		TenantID:    tenant,
-		TokenGetter: getToken.GetToken,
+		Host:                    host,
+		Port:                    uint32(port),
+		TenantID:                tenant,
+		TokenGetter:             getToken.GetToken,
+		EnvironmentURL:          cfg.GetEnvironmentURL(),
+		FetchOnStartupPageSize:  cfg.GetFetchOnStartupPageSize(),
+		FetchOnStartupRetention: cfg.GetFetchOnStartupRetention(),
```

Comment:
These 2 config can probably be applied to streamer instead as watch manager itself will not use these config.

Reply:
moved